### ISSUE TITLE
Don't crash if table doesn't exist

### ISSUE THREAD

**Issue by jabdr** [jabdr/sql_query#1]
Currently sqlalchemy just throws an exception. We should just check and fail if the table does not exist.

**Comment by jabdr** [comment]
```python
def table_exists(name):
    ret = engine.dialect.has_table(engine, name)
    print('Table "{}" exists: {}'.format(name, ret))
    return ret
```

**Comment by jabdr** [comment]
Fixed in 4938b7e16bbdf5bc63e19954405833ebd6307d93
